WorkloadManagementSystem/JobWrapper/Watchdog.py

```diff
@@ -940,9 +950,17 @@ def getMemoryUsed(self):
 
   #############################################################################
   def getDiskSpace(self):
-    """ Attempts to get the available disk space, should be overridden in a subclass"""
-    methodName = 'getDiskSpace'
-    self.log.warn('Watchdog: ' + methodName + ' method should be implemented in a subclass')
-    return S_ERROR('Watchdog: ' + methodName + ' method should be implemented in a subclass')
+    """Obtains the disk space used.
+    """
+    result = S_OK()
+    diskSpace = getDiskSpace()
+
+    if diskSpace == -1:
+      result = S_ERROR('Could not obtain disk usage')
+      self.log.warn(' Could not obtain disk usage')
+      result['Value'] = -1
```

Comment:
This didn't change, but this value is just overwritten in line 963 (with `float(-1)` if this branch is taken, so nothing wrong at the moment, but it smells), the line could just be removed, or a `return result` after this line to be independent of what happens after the if.
```suggestion
      result['Value'] = float(-1)
      return result
```

Reply:
Implemented this and the other suggestions.